api/src/test/java/org/openmrs/BaseFormRecordableOpenmrsDataTest.java

```diff
@@ -0,0 +1,140 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+import liquibase.util.StringUtils;
+import org.junit.Assert;
```

Comment:
please replace the JUnit 4 assertions in this test `import org.junit.Assert` since we migrated to JUnit 5. JUnit 5 ones are in the Jupiter package https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html